alembic/versions/fe7656c21eaa_add_remainder_of_database_models.py

```diff
@@ -0,0 +1,67 @@
+"""add remainder of database models
+
+Revision ID: fe7656c21eaa
+Revises: 5344fc3a3d3f
+Create Date: 2018-08-01 15:44:29.298131
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'fe7656c21eaa'
+down_revision = '5344fc3a3d3f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'sources',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('uuid', sa.String(length=36), nullable=False),
+        sa.Column('journalist_designation', sa.String(length=255),
+                  nullable=False),
+        sa.Column('is_flagged', sa.Boolean(), nullable=True),
```

Comment:
i like the join table with possible tag values pattern (because normalization), let's do that when we add more tags. btw as background here are some notes about these columns:

* `is_flagged`: this column tracks if SecureDrop's flag for reply functionality has been enabled for that source. This flag for reply functionality is a DoS mitigation (it prevents the generation of source keys if many sources are created in a limited time period). Check out [these screenshots](https://docs.securedrop.org/en/stable/journalist.html#flag-for-reply) to see how this looks from the journalist's perspective

* `is_starred`: this is a system-wide star - so SecureDrop starring is like pinned messages on Slack, which is a little confusing imho (this is legacy behavior) - users might expect this to be per-user.